usr.bin/bsdiff/bspatch/bspatch.c

```diff
@@ -151,7 +152,7 @@ int main(int argc, char *argv[])
 	if (cap_enter() < 0)
 		err(1, "failed to enter security sandbox");
 
-	cap_rights_init(&rights_ro, CAP_READ, CAP_FSTAT, CAP_SEEK);
+	cap_rights_init(&rights_ro, CAP_READ, CAP_FSTAT, CAP_SEEK, CAP_MMAP_R);
```

Comment:
I think a more faithful use of capsicum here would be to add a new rights_mmap_r that only includes `CAP_SEEK` and `CAP_MMAP_R` leaving `rights_ro` unchanged and using the new permission for `oldfd`.  Perhaps that doesn't matter in this case though?

Reply:
> I think a more faithful use of capsicum here would be to add a new rights_mmap_r that only includes `CAP_SEEK` and `CAP_MMAP_R` leaving `rights_ro` unchanged and using the new permission for `oldfd`. Perhaps that doesn't matter in this case though?

We're only mmap'ing the input file, not the patch file.